src/main/java/com/mael/ttt/Board.java

```diff
@@ -23,6 +23,13 @@ public Board (Board board) {
         }
     }
 
+    public Board(Mark ... cellContents) {
```

Comment:
If that's a constructor that is only used in tests, it might be worthwhile to have it scoped to the package.
Of course, that's only if it is not used/needed in the production code.